skeletons/INTEGER.h

```diff
@@ -43,6 +43,8 @@ xer_type_encoder_f INTEGER_encode_xer;
 per_type_decoder_f INTEGER_decode_uper;
 per_type_encoder_f INTEGER_encode_uper;
 
+#define INTEGER_constraint asn_generic_no_constraint
```

Comment:
> So your code works fine . . . is very reasonable, and my #143 works too

In that case and at this time I'd rather avoid a simplification whose consequences I don't fully understand.

> (1) Why `asn_DEF_INTEGER` and `asn_DEF_NativeInteger` has `asn_generic_no_constraint` as their `asn_constr_check_f` field ?

Good question. We should hope to find an answer eventually.

> (2) Why the following ASN.1 statement does not generate constraint check function ?
```
Version  ::=  INTEGER  {  v1(0), v2(1), v3(2)  }
```

This is a good question that I think we must find an answer to, and preferably sooner rather than later. It seems related to the issue described in https://github.com/vlm/asn1c/pull/129#issuecomment-286685144.